### PR TITLE
AWS- ACM add aws api module

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py [hunk 1]
```diff
@@ -326,35 +218,61 @@ def get_certificate(args):
...
+        timeout = demisto.params().get('timeout')
+        retries = demisto.params().get('retries') or 5
+
+        validate_params(aws_default_region, aws_role_arn, aws_role_session_name, aws_access_key_id,
+                        aws_secret_access_key)
+
+        aws_client = AWSClient(aws_default_region, aws_role_arn, aws_role_session_name, aws_role_session_duration,
+                               aws_role_policy, aws_access_key_id, aws_secret_access_key, verify_certificate, timeout,
+                               retries)
+        args = demisto.args()
```
Comment (bakatzir) [inline Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py:248]: please call also demisto.command() once here.

File: Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py [hunk 2]
```diff
@@ -326,35 +218,61 @@ def get_certificate(args):
...
+            remove_tags_from_certificate(args, aws_client)
+        if demisto.command() == 'aws-acm-list-tags-for-certificate':
+            list_tags_for_certificate(args, aws_client)
+        if demisto.command() == 'aws-acm-get-certificate':
+            get_certificate(args, aws_client)
+
+    except ResponseParserError as e:
+        return_error('Could not connect to the AWS endpoint. Please check that the region is valid.\n {error}'.format(
+            error=type(e)))
+        LOG(str(e))
```
Comment (bakatzir) [inline Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py:267]: i think this is handeld already in the apimodule

File: Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py [hunk 3]
```diff
@@ -326,35 +218,61 @@ def get_certificate(args):
...
+
+    except Exception as e:
+        LOG(str(e))
+        return_error('Error has occurred in the AWS ACM Integration: {code}\n {message}'.format(
+            code=type(e), message=str(e)))
+
+
+from AWSApiModule import *  # noqa: E402
+
+if __name__ in ['__builtin__', 'builtins', '__main__']:
```
Comment (bakatzir) [inline Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py:277]:
```suggestion
if __name__ in ('__builtin__', 'builtins', '__main__'):
```

File: Packs/AWS-ACM/Integrations/AWS-ACM/README.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
+Amazon Web Services Certificate Manager Service (acm)
```
Comment (bakatzir) [inline Packs/AWS-ACM/Integrations/AWS-ACM/README.md:1]:
```suggestion
Amazon Web Services Certificate Manager Service (ACM)
```

File: Packs/AWS-ACM/ReleaseNotes/1_1_0.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### AWS - ACM
+- Internal code improvements.
```
Comment (bakatzir) [inline Packs/AWS-ACM/ReleaseNotes/1_1_0.md:4]: can rm. not needed when we have detailed rn. any pr contains also some internal code improvements..

File: Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py (hunk 2 above)
Comment (tallieber) [reply Packs/AWS-ACM/Integrations/AWS-ACM/AWS-ACM.py:267]: We don't check at the api module if the region is valid, we just require from the user to fill it, so I think we should leave it here

**Review by bakatzir — APPROVED** [review]

**Pull request merged**